docs/pages/product/workspace/encryption-keys.mdx

```diff
@@ -0,0 +1,89 @@
+# Encryption keys
+
+The <Btn>Encryption Keys</Btn> page in Cube Cloud allows to manage [data-at-rest
+encryption in Cube Store][ref-cube-store-encryption].
+
+<SuccessBox>
+
+Data-at-rest encryption in Cube Store is only available in Cube Cloud on
+[Enterprise and above plans](https://cube.dev/pricing).
+
+</SuccessBox>
+ 
+Navigate to <Btn>Settings → Encryption Keys</Btn> in your Cube Cloud deployment
+to [provide](#add-a-key), [rotate](#rotate-a-key), or [drop](#drop-a-key)
+your own customer-managed keys (CMK) for Cube Store.
+
+## Customer-managed keys for Cube Store
+
+On the <Btn>Encryption Keys</Btn> page, you can see all previously provided keys:
+
+<Screenshot src="https://ucarecdn.com/48038ac1-fdf1-4c87-8860-ac503bfcdac3/" />
+
+### Add a key
+
+To add an encryption key, click <Btn>Create</Btn> to open a modal window.
+Provide any string as the key name and paste a [Base64-encoded][link-base64]
```

Comment:
It specifically has to be standard base64 in its canonical representation.  I think it is a good idea to say it has to be standard base64, and change the link to specifically point to something defining standard base64.  I like how this Google documentation does so: https://cloud.google.com/storage/docs/encryption/customer-supplied-keys

Just to be clear, we actually do reject base64 that isn't in its canonical representation -- we check `x == encode(decode(x))`.  The UI provides instant feedback to the user if this is not the case.